### ISSUE TITLE
Create table if table doesn't exist.

### ISSUE THREAD

**Issue by BirkHappens** [paneio/happyhour#1]
On first run, if table doesn't exist, we should create the table.

**Comment by rynmln** [comment]
Agreed, it should also create the data directory for the db as well. I'd say this is more of a bug since it's not immediately obvious that the code doesn't do this for you until first run.
